assets/src/apps/authoring/components/PropertyEditor/custom/CustomTextbox.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useEffect, useState } from 'react';
+
+interface CustomTextboxProps {
+  label: string;
+  id: any;
+  value: string;
+}
+const CustomTextbox: React.FC<CustomTextboxProps> = (props: any) => {
```

Comment:
```suggestion
const CustomTextbox: React.FC<CustomTextboxProps> = (props) => {
```
the generic props definition already passed in so props will have that `CustomTextboxProps` type already